### ISSUE TITLE
feat(patch) Instagram patches to make the app less addictive

### ISSUE THREAD

**Issue by holy-crusader-py** [ReVanced/revanced-patches#2879]
### Application

Instagram 

### Issue

Instagram is to addictive and like other applications, it draws us into a loop of infinite content.

 A mod exists called Distraction Free Instagram, but it's really messed up

### Patch

Here are the patches I propose:

- An option to remove reels suggestions from the feed
- Remove the ability to scroll when viewing a reel
- Remove infinite suggestions from search page
- Remove suggested content when you've finished viewing everything you're following
- A button to prevent these parameters from being changed

### Motivation

We waste too much time looking at useless content suggested by Instagram, the app does everything to draw us into an endless loop of infinite content and this useless content could be removed from the App to make the experience less addictive, less useless and what it's supposed to be

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by holy-crusader-py** [comment]
It's easier to stop yourself from doing something than to tell yourself you're not going to do it.

It's a feature on Distraction Free Instagram that I don't use, but I've seen people using it, so I thought it would be useful.

**Comment by mario-g98** [comment]
Yes, please. I just hate the infinite suggestions on the home feed.

**Comment by GrayLanterns** [comment]
Do patching even work on Instagram anymore? Revanced does not hide ads for me on Instagram, like it says it does.

**Comment by holy-crusader-py** [comment]
Depends on the version you are patching and there are many types of ads on Instagram
